modules/search/class.jetpack-search.php

```diff
@@ -1440,21 +1440,6 @@ public function set_filters( array $aggregations ) {
 		$this->aggregations = $aggregations;
 	}
 
-	/**
-	 * Set the search's facets (deprecated).
-	 *
-	 * @deprecated 5.0 Please use Jetpack_Search::set_filters() instead.
-	 *
-	 * @see        Jetpack_Search::set_filters()
-	 *
-	 * @param array $facets Array of facets to apply to the search.
-	 */
-	public function set_facets( array $facets ) {
```

Comment:
@nickdaugherty do you think this is safe for all VIPs to have removed? My guess is yes. I think it is a holdover from the old WPCOM_elasticsearch code, but maybe worth searching through all VIP repos. I don't see anything on wpcom.

Reply:
With GH's search I was able to find 1 instance of a VIP site using this directly, we're doing a more comprehensive check now.

Reply:
Looks like this is indeed the only instance, we're working with the customer to get it switched.

Reply:
Thanks @nickdaughtery. In terms of timeline, the second Tuesday of January is when this version would go stable (presuming it lands this month). We're pushing a week later due to the holiday season.

Reply:
Thanks @kraftbj - we'll be all set on our end by that date, so 👍 from me on this change.